util/saveItemToRemarkable.ts

```diff
@@ -0,0 +1,54 @@
+import cheerio from "cheerio";
+import type { Remarkable } from "remarkable-typescript";
+import pdf from "html-pdf";
+import { v4 as uuidv4 } from "uuid";
+
+const elementsToRemove = [
+  "script",
+  "footer",
+  "head",
+  "svg",
+  "nav",
+  "input",
+  "button",
+  "img",
+  "*[class*=menu]",
+  "*[class*=navigation]",
+  "*[class*=nav]",
+  "*[class*=sidebar]",
+  "*[class*=recommendation]",
+  "*[class*=newsletter]",
+  "*[class*=cookie]",
+];
+
+const pdfOptions = {
+  border: "1cm",
+};
+
+export const saveItemToRemarkable = async ({
+  item,
+  remarkableClient,
+  folderId,
+}: {
+  item: any;
+  remarkableClient: Remarkable;
+  folderId: string;
+}) => {
+  const response = await fetch(item.link);
+  const rawHtml = await response.text();
+  const $ = cheerio.load(rawHtml);
```

Comment:
not sure if the html cleaning part should also be an own function, as it's technically not implied by this functions name "saveItemToRemarkable".